docs/platforms/kubernetes/deployment-logs.md

```diff
@@ -0,0 +1,58 @@
+# Kubernetes event logs
+
+To stream Kubernetes events into your workspace startup logs, you can use Coder's [`coder-logstream-kube`](https://github.com/coder/coder-logstream-kube) tool. `coder-logstream-kube` provides useful information about the workspace pod or deployment, such as:
+
+- Causes of pod provisioning failures, or why a pod is stuck in a pending state.
+- Visibility into when pods are OOMKilled, or when they are evicted.
+- Filter by namespace, field selector, and label selector to reduce Kubernetes API load.
```

Comment:
What do you mean by this?

Reply:
Copied this from the logstream-kube repo.

Reply:
The third line is more of a feature than what kind of visibility it gives users. Let's remove it from here.